package.json

```diff
@@ -81,6 +85,7 @@
     "@kiwicom/orbit-components": "^0.0.0-rc1",
     "axios": "^0.18.0",
     "babel-runtime": "^6.26.0",
+    "graphql": "0.13.2",
```

Comment:
how is this related to this PR?

Reply:
I have no idea why this is hightlighted. AFAIK `graphql` package should have been in `package.json` before this PR was created. :confused:

Reply:
no, it's already in devDependencies, this looks more like an error

Reply:
You're totally right, missed it.